docs/Team/index.md

```diff
@@ -36,10 +48,19 @@ All developers will be able to perform code reviews and give constructive feedba
 The rest of these are more what you call [guidelines, not actual rules](https://youtu.be/jl0hMfqNQ-g).
 
 ## Advancement
-You've heard it said that you should dress for the job you want, well similarly, if you want to be a Senior or a Lead developer, you should already be demonstrating those traits. In other words if you ask for a promotion, it shouldn't cause your manager to hesitate.
+You've heard it said that you should dress for the job you want, well similarly, if you want to be a Senior Developer or a Manager, you should already be demonstrating the traits respective for those roles. In other words if you ask for a promotion, it shouldn't cause your manager to hesitate.
 
 You should also have a plan and some goals to be working towards, even if you are not looking for a promotion. Goals are tricky. Like estimating software in general, it's quite easy to overestimate how much you can accomplish in a year's time. And in our business, client work has a tendency to take up as much free time as possible, so while special consideration will be given if it's been a particularly busy time, we need to strive to make forward progress.
 
+### Snowflake
+Medium has a great tool for measuring career performance called Snowflake. It measures technical as well as interpersonal skills. We liked it so much we adopted it for the dev team at Tallwave. [You can view the tool itself over here](https://tallwave.github.io/snowflake). 
+
+Snowflake is not currently tied to any *official* policies at Tallwave - it is more for feedback and performance planning. There are 16 Tracks, each with 5 tiers. Advancement through tiers gets progressively harder. The difference between a 4 and a 5 is significantly larger than between 1 and 2. The left most tracks are more technical in nature, and the right most are more people & relationship based. Individual Contributors will tilt towards the left and Managers towards the right, though a mix between the two is normal.
+
+There will be a tension between what people are capable of doing versus what they are already demonstrating. A person should only be moved up a tier if they have consistently shown that they meet the criteria at that tier. And it should be noted the example behaviors and tasks are **examples** and not checklists.
```

Comment:
The use 'tension' here needs clarification. Does it imply 'discrepancy', 'one-to-one relationship', 'tendencies', etc?

Reply:
As in it can be uncomfortable discussing this. There is a clear difference between "I think you can do X" vs "You have demonstrated that you can do X consistently". We've (*I've*) gotten in trouble by not understanding that.

Reply:
👍